app/assets/stylesheets/dashboard.scss

```diff
@@ -27,6 +27,10 @@
     font-style: italic;
   }
 
+  #admin-dropdown {
```

Comment:
Rather than add a custom style here, can you add the bootstrap class `mb-4` (or similar) to the element's markup?

Reply:
Oh duh, yes, will do.